core/src/main/java/hudson/util/TimeUnit2.java

```diff
@@ -63,7 +67,12 @@
  *
  * @since 1.5
  * @author Doug Lea
+ * @deprecated use {@link TimeUnit}. (Java 5 did not have all the units required, so TimeUnit2 was introduced because it
+ * had better conversion until Java 6 went out.)
  */
+@Deprecated
+@RestrictedSince("TODO")
+@Restricted(DoNotUse.class)
```

Comment:
I generally reserve restriction for newly introduced (or modified) APIs, or cases where ongoing use is clearly going to break, so this seems like overkill. I am not objecting, just FYI.